ci/diff-with-previous.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
on my ubuntu system `/bin/sh` is a link to `/bin/dash` which fails to run this. Any reason why we still use `sh` instead of just `bash`?

Reply:
No, that's a good point. I try to be explicit about using `bash` but I don't always remember!